src/main/java/mat/server/CQLLibraryService.java

```diff
@@ -502,6 +509,43 @@ public void saveCQLLibraryExport(CQLLibrary cqlLibrary, String cqlXML) {
         cqlLibraryExportDAO.save(cqlLibraryExport);
     }
 
+    @Override
+    public CheckForConversionResult checkLibraryForConversion(CQLLibraryDataSetObject sourceLibrary) {
+        logger.info("checkLibraryForConversion  libraryId: " + sourceLibrary.getId() + " setId: " + sourceLibrary.getCqlSetId());
+        CheckForConversionResult result = new CheckForConversionResult();
+        List<CQLLibrary> draftLibraries = cqlLibraryDAO.getDraftLibraryBySet(sourceLibrary.getCqlSetId());
+        if (draftLibraries.isEmpty()) {
+            // If no drafts found we can proceed with conversion
+            result.setProceedImmediately(true);
+        } else {
+            // If the only draft is a FHIR draft created from the same source measure then ask for confirmation to override.
+            // UI cannot proceed with conversion if User didn't confirm or if there is/are other draft(s)
+            //TBD
```

Comment:
TBD - probably you meant to add something here

Reply:
I had one but forgot to remove the comment.